README.md

```diff
@@ -1,2 +1,67 @@
 # consert-website
-A website consert
+
+
+ This is a Microverse consert website project.
+ 
+## Built With
+
+- HTML
+- css
+- javascript
+## Live Demo (not available)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+A web browser (e.g. Mozilla Firefox, Google Chrome etc.)
+
+git: This is a version control system used for source code management.
+
+A text editor (e.g Visual Studio Code (recommended), Sublime Text, Atom etc.)
+
+Live Server: A web extension utility for Visual Studio Code that helps you live reload feature for dynamic pages.
+
+Basic knowledge on HTML Javascript and CSS.
+
+### Setup
+With git, clone the code to your machine, or download the files directly.
+
+git clone https://github.com/leonard33/consert-website.git
+
+Once the files are on your machine, open the consert website folder in Visual Studio Code.
+
+### Deployment
+After opening the files in Visual Studio Code, press the Go Live button on the status bar (bottom of the window) to launch the files with Live Server.
+
+
+
+## Authors
+
+👤 **Author1**
+  Leonard Kombo
+- GitHub: [@githubhandle](https://github.com/leonard33)
+- Twitter: [@twitterhandle](https://twitter.com/leonardkombo1)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/leonardkombo)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] Your README should contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design], Kindly add that